README.rst

```diff
@@ -181,6 +181,4 @@ License
    :target: https://gitter.im/coala/coala
 .. |AGPL| image:: https://img.shields.io/github/license/coala/coala-bears.svg
    :target: https://www.gnu.org/licenses/agpl-3.0.html
-.. |Throughput Graph| image:: https://graphs.waffle.io/coala/coala-bears/throughput.svg
-   :target: https://waffle.io/coala/coala-bears/metrics/throughput
-   :alt: 'Throughput Graph'
```

Comment:
Few missing things:
- Reason for change/removal?
- No good commit message.
- Bad PR title.
- No useful description in the PR.